gematria/granite/graph_builder_model_inference.h

```diff
@@ -87,10 +88,22 @@ class GraphBuilderModelInference {
   // structure of a model based on the BasicBlockGraphBuilder class.
   GraphBuilderModelInference(
       std::unique_ptr<BasicBlockGraphBuilder> graph_builder,
-      const tflite::FlatBufferModel* tflite_model);
+      const tflite::FlatBufferModel* tflite_model,
+      std::unique_ptr<tflite::Interpreter> interpreter,
+      std::unique_ptr<std::vector<int>> input_tensor_to_idx,
+      bool uses_deltas = true);
```

Comment:
We could take it the other way around, and detect `use_deltas` from the model (if there is the delta block mapping, then it uses deltas; otherwise, it doesn't use them).

If we can't auto-detect `use_deltas`, then I'd prefer to have it provided explicitly to avoid any errors where it is skipped by accident.

Reply:
Ah, `FromTfLiteModel` checks for the delta block index tensor and passes `uses_deltas` to the `GraphBuilderModelInference` constructor accordingly. I should remove the default value to avoid confusion.